atmo/coordinator/capabilities/capabilities.go

```diff
@@ -0,0 +1,75 @@
+package capabilities
+
+import (
+	"github.com/pkg/errors"
+	"github.com/suborbital/atmo/atmo/appsource"
+	"github.com/suborbital/reactr/rcap"
+	"github.com/suborbital/vektor/vlog"
+)
+
+// Render "renders" capabilities by layering any user-defined
+// capabilities onto the default set, thus allowing the user to omit any
+// individual capability (or all of them) to receive the defaults
+func Render(source appsource.AppSource, log *vlog.Logger) (rcap.CapabilityConfig, error) {
```

Comment:
This should take in the 'base' capabilities as an arg, because in Sat we create the defaults outside of this function.

Reply:
Ah right.